js/src/traceable.ts

```diff
@@ -293,16 +314,35 @@ export function traceable<Func extends (...args: any[]) => any>(
     getInvocationParams?: (
       ...args: Parameters<Func>
     ) => InvocationParamsSchema | undefined;
+
+    /**
+     * Apply transformations to the inputs before logging.
```

Comment:
_nit_: would add a disclaimer that the `processInputs` is not inherited in inner `traceable` calls